c2/c2-client-bundle/c2-client-service/src/test/java/org/apache/nifi/c2/client/service/operation/TransferDebugOperationHandlerTest.java

```diff
@@ -87,6 +91,18 @@ public class TransferDebugOperationHandlerTest {
     @TempDir
     private File tempDir;
 
+    @AfterAll
+    public static void cleanUpAfterAll() {
```

Comment:
It seems like this could be better as an `AfterEach` method. Ideally, the Handler class would have the temporary directory as a parameter, but given the usage of createTempDirectory(), this style of expecting the system temporary seems reasonable.